flake.nix

```diff
@@ -0,0 +1,19 @@
+{
+  description = "Intel® NPU driver";
+
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+
+  outputs = { self, nixpkgs, ... } @ inputs:
```

Comment:
can you also add the flake.lock file along with the flake.nix file? It would be awesome to have the reproducibility.